.platform/hooks/prebuild/02_build_frontend.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+source ~/.nvm/nvm.sh
+npm --prefix src/ts run build
```

Comment:
doesn't webpack do something like this automatically? Or is this something different?

Reply:
This invokes webpack through a script, and webpack places the build in the `static` folder of our flask app. We have to invoke it though. Webpack and Node do not need to run in production.